### PR TITLE
MAINT: Assorted clean up.

### PR DESCRIPTION
**Pull request by WarrenWeckesser**

### CODE REVIEW

File: scipy/constants/codata.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
 from __future__ import division, print_function, absolute_import
 
 import warnings
-from math import pi, sqrt
+from math import pi
```
Comment (pvanmulbregt) [inline scipy/constants/codata.py]: sqrt symbol will be needed again in gh-11345, so I left it in and deleted gh-11537.

**Comment by pvanmulbregt** [comment]
I was doing final round of cleanup for gh-11529 and you've identified some of the same ones I found along with some formatting cleanup. Apart from leaving the sqrt in codata.py it looks good.

File: scipy/constants/codata.py (hunk 1 above)
Comment (WarrenWeckesser) [reply scipy/constants/codata.py]: OK, I've removed this change in my latest update.

**Comment by WarrenWeckesser** [comment]
I force-pushed an update with the change of `sqrt` in "constants/codata.py" removed, and with a few more PEP 8 fixes and docstring tweaks.

**Comment by rgommers** [comment]
All green, merged. Thanks @WarrenWeckesser and @pvanmulbregt

**Pull request merged**